src/uniprotkb/components/protein-data-views/PtmExchangeEvidenceTag.tsx

```diff
@@ -103,7 +103,7 @@ const PtmExchangeEvidenceTag = ({
     >
       <section className={style['evidence-tag-content']}>
         <h5 data-article-id={`evidences#${originalEvidenceCode}`}>
-          {originalEvidenceData.label}{' '}
+          {originalEvidenceData.label(originalEvidences)}{' '}
           <small>({originalEvidenceData.description})</small>
```

Comment:
Is there still a case where we render both? I thought we now never had the case of a heading saying something with small text saying something similar

Reply:
I missed to remove this somehow. Right we don't show both of them anywhere